### DISCUSSION TITLE
Developing a language?

### DISCUSSION THREAD

**Discussion by Magnus-Wizard** [community/community#63300] [Programming Help] [answered]
### Body

Okay
Hello everyone and thanks for last time...(s) 😅 . Your answers are always helpful and so I just thought it would be nice to mention this.

I have been having a tough time lately, trying to comprehend creating a language. I have a brilliant idea for a programming language and I mean it's like very bright 😂
But so far I don't understand how it's done. I've been through a series "you would use Java or c++" and then there is assembly something and all and then I'm told I need to write a lexer or something but I would need to use c to write it, which makes me wonder how c was made in the first place and it seems like it's all one big endless circle
I would have attempted to make the language with python but I just want to start from the beginning or at least know about it and I was told the language would be slow if I use python
This is probably getting long 😅
My point is that if there were no other languages how would I make mine? 

Edit:
I did more reading and now I'm totally lost 💔 please explain like I'm five
I know python and js only
### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by elderguardian** [comment, marked as answer]
It does not matter, just start

**Comment by Magnus-Wizard** [comment]
So I did a bit of research and I understand well now
Lol I'm starting... Thanks
